changelog.d/11867.feature

```diff
@@ -0,0 +1 @@
+Stabilize MSC3231.
```

Comment:
I would additionally add a note to https://github.com/matrix-org/synapse/blob/develop/docs/upgrade.md, as sysadmins may need to update their reverse proxy configuration.